ajax/libs/spritejs/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "spritejs",
+  "description": "A cross platform high-performance graphics system.",
+  "filename": "spritejs.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/spritejs/spritejs.git"
+  },
+  "author": "akira-cn",
+  "homepage": "http://spritejs.org",
+  "keywords": ["spritejs", "canvas", "graphics", "rendering", "render", "svg-path", "svg", "webgl2", "webgl", "3d", "2d"],
+  "license": "MIT",
+  "npmName": "spritejs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "spritejs.js",
+        "spritejs.min.js",
+        "spritejs.es.js",
+        "spritejs.es.min.js",
+        "spritejs.worker.js",
+        "spritejs.worker.min.js",
+        "spritejs.worker.es.js",
+        "spritejs.worker.es.min.js"
```

```suggestion
        "spritejs.*"
```
This is loading every spritejs file from dist, so we can replace this with a single-line glob.

---

wow~ that's awesome, I didn't know we can write like this, thanks~